v03_pipeline/lib/tasks/write_project_family_tables.py

```diff
@@ -26,27 +27,26 @@ def complete(self) -> bool:
             for write_family_table_task in self.dynamic_write_family_table_tasks
         )
 
-    def run(self):
-        # https://luigi.readthedocs.io/en/stable/tasks.html#dynamic-dependencies
-        # Fetch family guids from project table
-        update_project_table_task: luigi.Target = yield self.clone(
-            UpdateProjectTableTask,
-        )
-        project_ht = hl.read_table(update_project_table_task.path)
-        family_guids_in_project_table = set(hl.eval(project_ht.globals.family_guids))
+    def requires(self) -> list[luigi.Task]:
+        return [
+            self.clone(
+                WriteRemappedAndSubsettedCallsetTask,
+            ),
+            self.clone(
+                UpdateProjectTableTask,
+            ),
+        ]
 
-        # Fetch family guids from pedigree
-        pedigree_ht_task: luigi.Target = yield RawFileTask(self.project_pedigree_path)
-        pedigree_ht = import_pedigree(pedigree_ht_task.path)
-        families_guids_in_pedigree = {
-            f.family_guid for f in parse_pedigree_ht_to_families(pedigree_ht)
-        }
-
-        # Intersect them
-        family_guids_to_load = (
-            family_guids_in_project_table & families_guids_in_pedigree
+    def run(self):
+        ht = hl.read_matrix_table(
+            remapped_and_subsetted_callset_path(
```

Comment:
just sanity checking that this works for AnVIL loading where we don't have remaps (or usually subsets)

Reply:
yeah the same file gets generated, we just run a "subset" that has all families in it.